stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -545,6 +545,15 @@ public class MessageListView : ConstraintLayout {
         super.onDetachedFromWindow()
     }
 
+    /**
+     * Returns the inner [RecyclerView] that is used to display a list of message list items.
+     *
+     * @return The inner [RecyclerView] with messages.
+     */
+    public fun getInnerMessageList(): RecyclerView {
```

Comment:
Perhaps `getRecyclerView` for the method name? As a customer, I don't know what I'd expect to be the "inner message list".

Reply:
👍